templates/kubernetes/terraform/environments/prod/main.tf

```diff
@@ -75,13 +75,22 @@ module "kubernetes" {
   ]
 
   domain_name                 = local.domain_name
-  auth_enabled                = <% if eq (index .Params `userAuth`) "yes" %>true<% else %>false<% end %>
-  auth_domain                 = "auth.${local.domain_name}"
-  backend_service_domain      = "<% index .Params `productionBackendSubdomain` %>${local.domain_name}"
-  frontend_service_domain     = "<% index .Params `productionFrontendSubdomain` %>${local.domain_name}"
-  # This domain or address must be verified by the mail provider (Sendgrid, SES, etc.)
-  user_auth_mail_from_address = "noreply@${local.domain_name}"
-
+  user_auth = <% if eq (index .Params `userAuth`) "yes" %>[
```

Comment:
How about just changing the variable to default to `[]` so we don't have to have the empty list here?